app/assets/javascripts/components/tag_groups/ColorPick.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import reactCSS from 'reactcss'
+import { TwitterPicker } from 'react-color'
+
+export default class ColorPick extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      displayColorPicker: false,
+      color: this.props.color
+    };
+
+    this.handleClick = () => {
+      this.setState({ displayColorPicker: !this.state.displayColorPicker });
```

Comment:
I think it's better to declare it in the class itself and just do the bind here. Like:

```js
constructor(props) {
  // ...
  this.handleClick = this.handleClick.bind(this);
  // ...
}

handleClick() {
  this.setState( ... );
}
```